felix/bpf-gpl/tc.c

```diff
@@ -1226,6 +1292,33 @@ static CALI_BPF_INLINE struct fwd calico_tc_skb_accepted(struct cali_tc_ctx *ctx
 		fib = false; /* Disable FIB because we want to drop to iptables */
 	}
 
+	if (CALI_F_TO_HEP && !skb_seen(skb)) {
+		CALI_DEBUG("Host accesses nodeport backend %x:%d\n",
```

Comment:
Not sure I follow; looks like there are several codepaths that lead here (via `goto allow`) and not all are nodeports?